src/Streamly/Internal/FileSystem/Event/Darwin.hs

```diff
@@ -505,7 +505,7 @@ readOneEvent = do
 
 watchToStream :: Watch -> Stream IO Event
 watchToStream (Watch handle _ _) =
-    S.catRights $ S.parseMany readOneEvent $ S.unfold FH.reader handle
+    S.rights $ S.parseMany readOneEvent $ S.unfold FH.reader handle
```

Comment:
We should remove the import of import qualified Streamly.Internal.Data.Stream.IsStream as S from this module as it is deprecated. and should use Stream.catRights here.